packages/braid-design-system/package.json

```diff
@@ -2,51 +2,166 @@
   "name": "braid-design-system",
   "version": "31.24.2",
   "description": "Themeable design system for the SEEK Group",
-  "main": "lib/components/index.ts",
-  "bin": {
-    "braid-upgrade": "./codemod/bin.js"
-  },
-  "sideEffects": [
-    "lib/themes/**/*",
-    "lib/css/atoms/**/*",
-    "lib/css/reset/**/*",
-    "themes/**/*",
-    "reset/**/*",
-    "playroom/**/*",
-    "*.css.ts"
-  ],
-  "scripts": {
-    "chromatic": "chromatic --storybook-build-dir ./dist-storybook --exit-zero-on-changes --exit-once-uploaded --auto-accept-changes master",
-    "eslint": "eslint --cache .",
-    "format": "eslint --cache --fix . && prettier --cache --write .",
-    "prettier": "prettier --cache --list-different .",
-    "storybook": "SKU_TELEMETRY=false sku storybook",
-    "copy-codemod": "rm -rf ./codemod/dist && cp -r ../codemod/dist ./codemod/dist",
-    "build-storybook": "SKU_TELEMETRY=false sku build-storybook",
-    "generate-icons": "node ./scripts/generateIcons.js",
-    "tsc": "tsc"
+  "homepage": "https://seek-oss.github.io/braid-design-system/",
+  "bugs": {
+    "url": "https://github.com/seek-oss/braid-design-system/issues"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/seek-oss/braid-design-system.git",
     "directory": "packages/braid-design-system"
   },
-  "author": "SEEK",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/seek-oss/braid-design-system/issues"
+  "author": "SEEK",
+  "sideEffects": [
+    "**/atoms/**",
+    "**/entries/reset.ts",
```

Comment:
Can I ask why the reset needs this explicit `entries` path?

Reply:
Because the site imports from `src`.